OIIO/WriteOIIO.cpp

```diff
@@ -381,6 +388,7 @@ WriteOIIOPlugin::WriteOIIOPlugin(OfxImageEffectHandle handle,
     , _bitDepth(NULL)
     , _quality(NULL)
     , _dwaCompressionLevel(NULL)
+    , _zipCompressionLevel(NULL)
```

Comment:
- `zipquality` TIFF compression level
- OpenEXR 3.1.3+ `zip` and `zips` EXR compression level